scripts/build/release.sh

```diff
@@ -168,19 +168,19 @@ touch CHANGES_NEXT_RELEASE
 # dev release sets 'latest' and not 'X.Y.Z-next"
 if [ "$BROKER_RELEASE" != "dev" ]
 then
-  sed "s/(https:\/\/readthedocs.org\/projects\/fiware-orion\/badge\/?version=latest)](http:\/\/fiware-orion.readthedocs.io\/en\/latest\/?badge=latest)/(https:\/\/readthedocs.org\/projects\/fiware-orion\/badge\/?version=$NEW_VERSION)](http:\/\/fiware-orion.readthedocs.io\/en\/$NEW_VERSION\/?badge=$NEW_VERSION)/" README.md > /tmp/README.md
+  sed "s/(https:\/\/img.shields.io\/readthedocs\/fiware-orion.svg)](https:\/\/fiware-orion.rtfd.io)/(https:\/\/img.shields.io\/readthedocs\/fiware-orion\/$NEW_VERSION.svg)](https:\/\/fiware-orion.rtfd.io\/en\/$NEW_VERSION\/)/" README.md > /tmp/README.md
 else
-  sed "s/(https:\/\/readthedocs.org\/projects\/fiware-orion\/badge\/?version=$currentVersion)](http:\/\/fiware-orion.readthedocs.io\/en\/$currentVersion\/?badge=$currentVersion)/(https:\/\/readthedocs.org\/projects\/fiware-orion\/badge\/?version=latest)](http:\/\/fiware-orion.readthedocs.io\/en\/latest\/?badge=latest)/" README.md > /tmp/README.md
+  sed "s/(https:\/\/img.shields.io\/readthedocs\/fiware-orion\/$currentVersion.svg)](https:\/\/fiware-orion.rtfd.io\/en\/$currentVersion\/)/(https:\/\/img.shields.io\/readthedocs\/fiware-orion.svg)](https:\/\/fiware-orion.rtfd.io)/" README.md > /tmp/README.md
 fi
 mv /tmp/README.md README.md
 
 # Adjust Readthedocs documentation link for GET /version response. Note that the procedure is not symmetric 
 # (like in version.h), as dev release sets 'master' and not 'X.Y.Z-next"
 if [ "$BROKER_RELEASE" != "dev" ]
 then
-  sed "s/https:\/\/fiware-orion.readthedocs.org\/en\/master\//https:\/\/fiware-orion.readthedocs.org\/en\/$NEW_VERSION\//" src/lib/common/defaultValues.h > /tmp/defaultValues.h
+  sed "s/https:\/\/fiware-orion.rtfd.io\//https:\/\/fiware-orion.rtfd.io\/en\/$NEW_VERSION\//" src/lib/common/defaultValues.h > /tmp/defaultValues.h
 else
-  sed "s/https:\/\/fiware-orion.readthedocs.org\/en\/$currentVersion\//https:\/\/fiware-orion.readthedocs.org\/en\/master\//" src/lib/common/defaultValues.h > /tmp/defaultValues.h
+  sed "s/https:\/\/fiware-orion.rtfd.io\/en\/$currentVersion\//https:\/\/fiware-orion.rtfd.io\//" src/lib/common/defaultValues.h > /tmp/defaultValues.h
 fi
```

Comment:
Se second pair of sed also seems to work fine:

At release time:

```
$ export NEW_VERSION=1.15.0
$ export currentVersion=1.14.0-next
$ echo '"doc" : "https://fiware-orion.rtfd.io/"' | sed "s/https:\/\/fiware-orion.rtfd.io\//https:\/\/fiware-orion.rtfd.io\/en\/$NEW_VERSION\//"
"doc" : "https://fiware-orion.rtfd.io/en/1.15.0/"
```

At the time of opening a new version devel cycle:

```
$ export currentVersion=1.15.0
$ echo '"doc" : "https://fiware-orion.rtfd.io/en/1.15.0/"' | sed "s/https:\/\/fiware-orion.rtfd.io\/en\/$currentVersion\//https:\/\/fiware-orion.rtfd.io\//"
"doc" : "https://fiware-orion.rtfd.io/"
```

NTC (just informative)